README.md

```diff
@@ -35,9 +32,23 @@ make test
 
 ### End to end tests
 
+The App Framework e2e tests in the App Framework project require the same Docker containers, used in the development environment step, to be running. However these tests don't need a Mattermost server to be running. The tests instead mimic the behavior of a running server using shared code of the `mattermost-server` repository. You can think of it as a "fake server" running, completely separate from the running containers, but communicating with the containers.
```

Comment:
This and the following paragraphs don't seem to fit under the "End to end tests" title. Should they be part of "Getting Started"?

Reply:
@hanzei This paragraph in particular does seem to be specific to e2e tests, but the paragraphs below should be in the "Getting Started" section.

Reply:
This paragraph seems to talk about the _Go_ end to end tests. Can this be made more clear?